cookbooks/cdo-varnish/test/integration/cloudfront/minitest/test_default.rb

```diff
@@ -0,0 +1,315 @@
+# Chef test suite for CloudFront distribution behavior.
+
+# Chef node attributes as constants
+DASHBOARD_PORT = 8080
+PEGASUS_PORT = 8081
+VARNISH_PORT = 80
+LOCALHOST = 'localhost'
+VARNISH_URL = "#{LOCALHOST}:#{VARNISH_PORT}"
+ENVIRONMENT = 'integration'
+CLOUDFRONT_URL = "https://#{ENVIRONMENT}-studio.code.org"
+
+# Setup the mock-server and ngrok
+puts 'setup'
+NGROK_PID = spawn('/usr/local/ngrok/ngrok start cdo --config=/home/kitchen/.ngrok2/ngrok.yml --log=stdout')
+PID = spawn('cd ~; java -jar mock.jar')
```

Comment:
It looks like mock.jar is written by a chef recipe.  Is this integration test something that can only be run on chef-manage instances (i.e. not my laptop?)